Muslim-Software-Developers/Musdev-fe
# Feat/programs and gallery

# Feature/Programs and Gallery Section

## Checklist

- [ ] Specs (if applicable)
- [ ] Visual (Screenshot / GIF / mp4) of new feature (if applicable)

## PR Description
UI Design of the Programs and Gallery section of the home page.
## How to Test

1. Run the repo locally
2. Go to the programs and gallery section
3. Check different views
4. Magic! 🪄

## Desktop View
<img width="1101" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/2c445fa3-308a-4baf-9315-36346cf497a6">

<img width="1068" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/5f9c0d7f-f184-4982-b858-d165bd2b7009">


## Tab View
<img width="587" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/2a6f4d8b-f663-4f96-98d0-2949cf3fdb32">

<img width="564" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/87bd064b-fcab-42f8-afaa-60991ce17911">



## Mobile View
<img width="299" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/ae374a49-15d0-4e7c-801a-9338e50ce742">

<img width="313" alt="image" src="https://github.com/Muslim-Software-Developers/Musdev-fe/assets/67792966/209a2864-b56f-4b29-b4b0-a1c5502fd4cd">
